docs/reference/esql/processing-commands/limit.asciidoc

```diff
@@ -7,3 +7,7 @@ The `LIMIT` processing command enables you to limit the number of rows:
 ----
 include::{esql-specs}/docs.csv-spec[tag=limit]
 ----
+
+If not specified, `LIMIT` defaults to `500`. A single query can't return more
+than 10,000 rows, regardless of the `LIMIT` value. {es} ignores `LIMIT` values
+exceeding `10000`.
```

Comment:
Thanks for the review and rewordings, @jrodewig.
I'll remove the last sentence, though, since it's not entirely correct: a minimum between user-provided and 10K will be chosen for the "final" `LIMIT` in a query, but "intermediary" ones (like those before or between "pipeline breakers", like aggs) are effective as user-provided.

Reply:
Makes sense — Thanks, @bpintea!